### DISCUSSION TITLE
Preview des Styles par défaut à jour

### DISCUSSION THREAD

**Discussion by David-GH** [AmauriC/tarteaucitron.js#699] [Q&A] [answered]
Bonjour, 

Ceci date de 2015 a priori... : https://codepen.io/AmauriC/pen/QwaxVP

Est-ce qu'il y aurait quelque chose de ce style qui soit disponible avec les nouveautés ?

Ça serait sympa d'avoir une visibilité de ce type sur l'ensemble de la solution qui soit à jour afin de pouvoir se créer son fichier css personnalisé plus simplement.

**Comment by AmauriC** [comment, marked as answer]
Une démo est dispo ici ; )
https://amauri.io/tarteaucitron.php?orientation=popup

**Reply by David-GH** [reply]
Parfait. Merci pour cette réponse super rapide.

_Bon, au final j'aurais pu intégrer moi même une version de ce type ; j'y avais pas pensé lol_

**Reply by Wafy-source** [reply]
@AmauriC existe t'il une nouvelle version de ce module car ton url ne fonctionne pas ?
